x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/Cast.java

```diff
@@ -111,4 +111,11 @@ public boolean equals(Object obj) {
     public String toString() {
         return functionName() + "(" + field().toString() + " AS " + to().sqlName() + ")#" + id();
     }
-}
\ No newline at end of file
+
+    @Override
+    public String name() {
```

Comment:
Isn't `super.name + "AS " + to().sqlName() enough? What's the last character in super.name() that needs removing?

LGTM otherwise.

Reply:
Your suggestion would give: `CAST(SUM(salary))AS DOUBLE`, so I need to add the AS .. before the final closing `)`.

Reply:
Got it - thanks.